backend/private-graph/graph/schema.graphqls

```diff
@@ -1326,6 +1326,9 @@ type Query {
 	workspace_admins_by_project_id(project_id: ID!): [WorkspaceAdminRole!]!
 	isIntegrated(project_id: ID!): Boolean
 	isBackendIntegrated(project_id: ID!): Boolean
+	clientIntegrationData(project_id: ID!): Session
+	serverIntegrationData(project_id: ID!): ErrorGroup
+	logsIntegrationData(project_id: ID!): LogsConnection
```

Comment:
nit, not important: i wouldn't infer that this would give me first session, error, log unless i saw the resolver implementation. could consider returning a `ClientIntegration, ...` that would have a `FirstSession: ...`

Reply:
Yeah, I think that's a good call. I ended up making an `IntegrationStatus` type that all these endpoints can use as their return type. Still need to figure out the logs check, but will handle in the next PR.